tests/apm/mod.rs

```diff
@@ -170,5 +180,5 @@ fn logging() {
 
     coll.drop().unwrap();
 
-    fs::remove_file("test_apm_log.txt").unwrap();
+    fs::remove_file("test_log.txt").unwrap();
```

Comment:
Makes sense to me for now. In an ideal future we would allow the client to take in a generic Write-implementing object and pass in a string-like object for testing purposes; if `with_log_file(str)` is part of the spec, it can be added as a utility method.